### PR TITLE
[MO] - [test-container] -> strimzi kafka container travis timeout bug (new installation of docker)

### PR DESCRIPTION
**Pull request by see-quick**
### Type of change

- Bugfix

### Description

This PR adding the longer timeout for starting kafka and zookeeper, which is needed for travis

### CODE REVIEW

**Comment by scholzj** [comment]
Im not sure we really want to invest the time into installing different Docker. If other TestContainers work with the default Docker, we should make our own work with it as well. That will ensure that it also works for all users.

**Comment by see-quick** [comment]
So currently the `TestContainers` supports the default version of docker! Hurrah...

File: test-container/pom.xml [hunk 1]
```diff
@@ -13,6 +13,10 @@
 
     <artifactId>test-container</artifactId>
 
+    <properties>
+        <skipTests>true</skipTests>
+    </properties>
```
Comment (scholzj) [inline test-container/pom.xml:18]: Why do we have this here?

File: test-container/pom.xml [hunk 2]
```diff
@@ -42,6 +46,22 @@
...
+                        <artifactId>junit-platform-surefire-provider</artifactId>
+                        <version>1.0.3</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.junit.jupiter</groupId>
+                        <artifactId>junit-jupiter-engine</artifactId>
+                        <version>5.0.3</version>
+                    </dependency>
+                </dependencies>
+            </plugin>
```
Comment (scholzj) [inline test-container/pom.xml:64]: Should this be using the versions frmo the main pom.xml? Or is there any reason to have the versions hardcoded here?

File: test-container/pom.xml (hunk 1 above)
Comment (see-quick) [reply test-container/pom.xml]: remnant of code...i will remove it

File: pom.xml [hunk 3]
```diff
@@ -121,8 +121,10 @@
         <json-path.version>4.1.1</json-path.version>
         <jupiter.version>5.6.2</jupiter.version>
         <junit.platform.version>1.6.2</junit.platform.version>
+        <junit-platform-surefire-provider.version>1.0.3</junit-platform-surefire-provider.version>
```
Comment (Frawless) [inline pom.xml:124]: Is there any reason to use this specific version? Latest is 1.3.2

Comment (see-quick) [reply pom.xml:124]: I do not think so. I will changed it.

**Pull request merged**